bin/install

```diff
@@ -47,7 +47,8 @@ get_cpu() {
 get_download_url() {
   local version="$1"
   local platform="$(get_arch)"
-  echo "https://storage.googleapis.com/kubernetes-release/release/v${version}/bin/${platform}/$(get_cpu)/kubectl"
+  #echo "https://storage.googleapis.com/kubernetes-release/release/v${version}/bin/${platform}/$(get_cpu)/kubectl"
```

Comment:
We can just remove this instead of commenting it out.

Reply:
Yes sorry that's an debug leftover